src/base/bittorrent/session.cpp

```diff
@@ -4524,14 +4527,15 @@ void Session::startUpTorrents()
                 {
                     recoveredCategories.insert(category);
                     isCategoryRecovered = true;
-                    LogMsg(tr("Inconsistent data is detected. Category '%1' is assigned to some torrent(s) but it doesn't exist in the configuration file."
-                              " Its settings will be reset to default.").arg(category), Log::WARNING);
+                    LogMsg(tr("Detected inconsistent data: category isn't registered to the configuration file."
```

Comment:
>category isn't registered **to** the configuration file

Are you sure it's correct?

Reply:
Or it can be replaced with "registered in", maybe @qbittorrent/bug-handlers can provide some insights.

Reply:
>Why not "category is missing from the configuration file."? 

Adopted, thanks. PR updated.